sites/public/styles/overrides.scss

```diff
@@ -349,10 +349,12 @@ a.button.is-primary:hover {
 }
```

Comment:
It looks like the mocks also add a box-shadow for .hero__button:

`box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);`